### ISSUE TITLE
Error fetching nearby places

### ISSUE THREAD

**Issue by fgnievinski** [commons-app/apps-android-commons#4726]
**Summary:** 

Error fetching nearby places. no such column: location_language (cod...

**Steps to reproduce:** 

Click on the icon for nearby location in the bottom menu.
Wait for the GPS location to fix.
Click on the button for searching in this area.

**System logs:**

```
Add logcat files here (if possible).

Need help? See https://github.com/commons-app/apps-android-commons/wiki/Getting-app-logs-from-Android-Studio
```

**Device and Android version:** 

Google Pixel 3a
Android 12
stock version
 
**Commons app version:** 

3.1.1~1c9267ca0

**Screen-shots:** 

![Screenshot_20211210-211158](https://user-images.githubusercontent.com/5018500/145656792-8336c4b7-e7dc-4102-98e8-dc244a05a1b7.png)

**Would you like to work on the issue?**

I'm not an Android programmer but I can help with testing.

**Comment by nicolas-raoul** [comment]
Thank you for reporting!

Did you recently switch to a different version of the app manually (for instance by installing an APK)?
Or did you just update normally using F-Droid or Play Store?

Also, did you recently change language settings of the OS or app?

Would you mind trying to erase the app's data (in Android settings)? Please let us know whether that fixes the issue or not. Thanks!

**Comment by fgnievinski** [comment]
yes, I had recently reconfigured Android's preferred language settings.

I only installed from the Google Play store. but I did try uninstalling and
reinstalling, which did l didn't help. I also tried the beta version, to no
avail.

but now clearing the app's storage solved the issue, thank you for your
help.

**Comment by nicolas-raoul** [comment]
Do you remember what sequence of operations led to the problem?

Or even better, would you know steps to reproduce the error?

That would help us a lot to fix the bug so that it does not happen to other people in the future :-)
Thanks a lot!

**Comment by fgnievinski** [comment]
the sequence of steps would be:
- install the Commons app
- open and close the app
- change Android language
- open the app again
- try to access the Nearby menu

**Comment by nicolas-raoul** [comment]
Strangely I could not reproduce the issue.
But please anyone else please try and let us know :-)
Thanks Felipe and all!

**Comment by mds08011** [comment]
I am also having issues with nearby places and have not made any recent changes I can think of.

![Screenshot_20211222-204759.png](https://user-images.githubusercontent.com/8010709/147189534-bd20d4ae-8daa-41df-8711-d06c4fc2ba27.png)

**Comment by nicolas-raoul** [comment]
@mds08011 If I understand correctly you have not modified any Commons app setting recently, and have not recently installed the app via APK? Would you mind posting a logcat that includes the stacktrace?

**Comment by mds08011** [comment]
Correct, I installed from the Google Play Store at least a few months ago and do not recall changing any settings.

I can try to get those longs but it may be a while.

**Comment by mds08011** [comment]
Actually, please disregard my previous comments. My own issue with Nearby was easily resolved after clearing the app cache and storage.

**Comment by Rishavgupta12345** [comment]
this issue is already solved by pull request #4742 , Please close this issue.

**Comment by nicolas-raoul** [comment]
@Rishavgupta12345 Not sure what you are talking about, #4742 is an issue, with no associated PR. Let's keep this open and close if nobody experiences the same during the next beta.

**Comment by Rishavgupta12345** [comment]
> @Rishavgupta12345 Not sure what you are talking about, #4742 is an issue, with no associated PR. Let's keep this open and close if nobody experiences the same during the next beta.

YES I mentioned the pull request wrong but actually, this issue is not reproducible

**Comment by Light-jetPilot** [comment]
Also having this issue. No changes to my phone or the app. Just opened the app for the first time in several months and nearby feature is no longer working.

I'm not an app developer, but it looks like some kind of database error.

Device and Android version:
Samsung Galaxy A51
Android version 12

App version: 
3.1.1~1c9267ca0

![Screenshot_20220525-152052_Commons](https://user-images.githubusercontent.com/106275798/170352631-2202e3a6-0785-4624-a77c-ddb67492e122.jpg)

**Comment by nicolas-raoul** [comment]
@Light-jetPilot Could you please try to uninstall then reinstall the app?

**Comment by Light-jetPilot** [comment]
> @Light-jetPilot Could you please try to uninstall then reinstall the app?

Uninstalled, restarted, and reinstalled. Same issue. Also am having issue #4742

**Comment by mds08011** [comment]
> 
> Uninstalled, restarted, and reinstalled. Same issue. Also am having issue #4742 
> 


@Light-jetPilot have you tried going to Settings>Apps>Commons>Storage & Cache and then clearing both the storage and cache memory? That resolved both issues in  my case.

**Comment by Light-jetPilot** [comment]
This does fix the issue. Odd that a clean install wouldn't, but I'm guessing that local database wasn't removed in the uninstall, so it still contained the deprecated column.
Thanks for your help

**Comment by nicolas-raoul** [comment]
If anyone see such an error again (in the beta or stable release), please let us know. Thanks all! :-)
